### ISSUE TITLE
ons-popover element leaks in DOM

### ISSUE THREAD

**Issue by zhuangperola** [OnsenUI/OnsenUI#750]
I am using ons-popover to show some information when one button is clicked. To make ons-popover closed by click outside. I set cancelable to it like this.

```
<ons-popover direction="down" modifier="android more" animation="none" cancelable>
    ...
</ons-popover>
```

So I want to remove ons-popover element from DOM, and wrote codes like this,

```
ons.createPopover("popover.html", {parentScope: $scope}).then(function (popover) {
                popover.show(event);
                popover.once('posthide', function () {
                    $timeout(function () { 
                        popover.destroy();
                    });
                });
            });
```

which caused another bug in console. I used setImmediate too which has the same result.

`Uncaught TypeError: Cannot read property 'attr' of null` 

https://github.com/OnsenUI/OnsenUI/issues/717

**Comment by zhuangperola** [comment]
Hi @frankdiox Could you have a check of this problem, thanks.

**Comment by frankdiox** [comment]
@zhuangperola Thanks for reporting!

I have tried this in 1.3.2, 1.3.4 and master branch but I haven't got any error in the console. The popover is well destroyed every time. I am using Chrome. Any other information you think it's relevant?

```
$scope.create = function(e) {
    ons.createPopover("popover.html", {parentScope: $scope}).then(function (popover) {
        popover.show('#button');
        popover.once('posthide', function () {
            $timeout(function () {
                popover.destroy();
            });
        });
    });
};
```

**Comment by zhuangperola** [comment]
Hi @frankdiox I have tried more and I think I know how to cause it.

The codes like this has no error in the console 

```
<ons-popover direction="down" modifier="android more" animation="none" cancelable>
    {{messageBox}}
</ons-popover>
```

But the codes like this will cause error in the console.

```
<ons-popover direction="down" modifier="android more" animation="none" cancelable>
    <ons-row align="left" style="height: 50px">
        <ons-col>
            {{messageBox}}
        </ons-col>
    </ons-row>
</ons-popover>
```

**Comment by frankdiox** [comment]
@zhuangperola Thanks again, now I got it. It's an easy fix actually :)

**Comment by frankdiox** [comment]
@zhuangperola The popover element has been adapted to CustomElements in master branch and this is not happening anymore. If you want to use 1.3.4 or older you can find the fix here #754

Thanks!
